R/validation-rules.R

```diff
@@ -122,3 +125,12 @@ infer_type <- function(orderings) {
 
   "unknown"
 }
+
+update_type <- function(type, probabilities) {
```

Comment:
Could you add a test for that bug fix or make it more obvious in the existing tests if it's already there?